client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dashboard-v2.tsx

```diff
@@ -97,6 +101,26 @@ export default function SitesDashboardV2() {
 		[ setSitesViewState ]
 	);
 
+	useEffect( () => {
+		let filterOptions: AgencyDashboardFilterOption[] = [];
+		const filterMap: AgencyDashboardFilterOption[] = [
+			'all_issues',
+			'backup_failed',
+			'backup_warning',
+			'threats_found',
+			'site_disconnected',
+			'site_down',
+			'plugin_updates',
+		];
+		for ( let i = 0; i < filterMap.length; i++ ) {
+			if ( sitesViewState.filters.some( ( filter ) => filter.value === i + 1 ) ) {
+				filterOptions = [ filterMap[ i ] ];
+			}
+		}
+
+		updateDashboardURLQueryArgs( { search: sitesViewState.search,  filter: filterOptions } );
+	}, [ sitesViewState ] );
```

Comment:
```
interface AgencyDashboardFilterMap {
		[ key: number ]: AgencyDashboardFilterOption;
	}
	const filtersMap: AgencyDashboardFilterMap = {
		1: 'all_issues',
		2: 'backup_failed',
		3: 'backup_warning',
		4: 'threats_found',
		5: 'site_disconnected',
		6: 'site_down',
		7: 'plugin_updates',
	};

	useEffect( () => {
		const filtersSelected = sitesViewState.filters.map( ( filter ) => filtersMap[ filter.value ] );

		updateDashboardURLQueryArgs( { filter: filtersSelected } );
	}, [ sitesViewState.filters ] );

	useEffect( () => {
		updateDashboardURLQueryArgs( { search: sitesViewState.search } );
	}, [ sitesViewState.search ] );
	```

Reply:
Noting that we chatted in Slack and this change I added is actually a regression as it's causing state issues (and a recurring console warning), so we'll follow up with that and additional changes shortly.